src/Library/WebServer.cs

```diff
@@ -10,6 +10,7 @@ public class WebServer
     {
```

Comment:
Let's rename "WebServer" into something that gives an idea regarding what the server actually does.